0. Introduction/0.2 Principles.md

```diff
@@ -39,54 +39,54 @@ Entanglements (vertical or horizontal) also prevent engineers in any system (esp
 
 ## 0.2.1 Rewritability
-Every system should be developed with the rewritability principle in mind. The principle dictates that our assumptions in the systems we develop has the high potentiality of being reexamined and probably reconsidered. Every system out there should be easily rewritable as a measure of adherence to a forever growing and changing business requirements.
+Every system should be developed with the rewritability principle in mind. The principle dictates that our assumptions in the systems we develop have the high potential of being reexamined and probably reconsidered. Every system should be easily rewritable as a measure of adherence to forever growing and changing business requirements.
 
-Rewritable code is code that is easy to understand, modify and fully rewrite. Rewritable code is extremely modular and autonomous it encourages engineers to evolve it with the least amount of effort and risk possible.
+Rewritable code is code that is easy to understand, modify and fully rewrite. Rewritable code is extremely modular and autonomous; it encourages engineers to evolve it with the least effort and risk possible.
 
-Rewritable code doesn't play tricks on the reader. There are no hidden dependencies, injected routines at runtime nor unknown prerequisites. It should be plug-n-play - fork, clone, build and run all it's tests successfully with no issues. 
+Rewritable code doesn't play tricks on the reader. It should be plug-n-play - fork, clone, build and run all its tests successfully with no issues. There are no hidden dependencies, injected routines at runtime, nor unknown prerequisites. 
 
 ## 0.2.2 Mono-Micro
-Building monolithic systems with modular mindset. Every flow should be built fully independent from other flows. For instance, we may build a monolithic system with microservice mindset. Which means that any flow can be extracted out of the system and turned into it's own microservice or lambda with the least amount of effort possible.
+Build monolithic systems with a modular mindset, with every flow fully independent from other flows. For instance, we may build a monolithic system with a microservice mindset, meaning that any flow can be extracted out of the system and turned into its own microservice or lambda with the least effort possible.
 
-This principle goes hand in hand with the concept of autonomous components at a higher-level where flows are also autonomous from their neighboring flow and their hosting system.
+This principle goes hand in hand with the concept of autonomous components at a higher level where flows are also autonomous from their neighboring flow and their hosting system.
 
 ## 0.2.3 Level 0
-Code must be understandable by an entry-level individual in the engineering craft. Our code base continues to live only based on how easy it is to understand by the mass majority of engineers in the industry. The mass-majority of engineers in our industry will always be those who are new in the craft.
```

Comment:
We can rephrase this but not remove it:
`Our code base continues to live only based on how easy it is`